feedly_streams_block.py

```diff
@@ -45,25 +45,22 @@ class FeedlyStreams(RESTPolling):
 
     auth_token = StringProperty(title='Authorization Token',
                                 default='[[FEEDLY_AUTHORIZATION_TOKEN]]')
+    user_id = StringProperty(title='User ID',
+                             default='[[FEEDLY_USER_ID]]')
     queries = ListProperty(FeedlyStream, title='Streams')
     lookback = TimeDeltaProperty(title='Lookback Period')
 
     def __init__(self):
         super().__init__()
-        self._user_id = None # Feedly user id is used in some requests.
+        self.user_id = None # Feedly user id is used in some requests.
```

Comment:
Remove this line. The `user_id` in line 48 is declaring the StringProperty as a class variable (which is the correct way to declare configurable properties in blocks), but this would override that.